src/pages/ThorChainLP/components/ReusableLpStatus/ReusableLpStatus.tsx

```diff
@@ -169,8 +173,12 @@ export const ReusableLpStatus: React.FC<ReusableLpStatusProps> = ({
         {assets.map((_asset, index) => {
           const amountCryptoPrecision =
             _asset.assetId === thorchainAssetId
-              ? confirmedQuote.runeCryptoDepositAmount
-              : confirmedQuote.assetCryptoDepositAmount
+              ? 'runeCryptoDepositAmount' in confirmedQuote
+                ? confirmedQuote.runeCryptoDepositAmount
+                : confirmedQuote.runeCryptoWithdrawAmount
+              : 'assetCryptoDepositAmount' in confirmedQuote
+              ? confirmedQuote.assetCryptoDepositAmount
+              : confirmedQuote.assetCryptoWithdrawAmount
```

Comment:
and here too?

Reply:
https://github.com/shapeshift/web/pull/6282/commits/426cf5fa7596a1e64d7672b5ef605468a53ece5f